### PR TITLE
Add partition key integration tests

### PR DESCRIPTION
**Pull request by brfrn169**
This PR adds partition key related integration tests. The integration tests check if all data types for partition keys are handled correctly. Please take a look!

### CODE REVIEW

File: core/src/main/java/com/scalar/db/storage/cosmos/ConcatenationVisitor.java [hunk 1]
```diff
@@ -99,14 +98,9 @@ public void visit(TextValue value) {
...
     value
         .get()
-        .ifPresent(
-            b -> {
-              // TODO Can we really convert bytes to string directly?
-              //      Maybe need to use the BASE64 encoding?
-              ByteBuffer buffer = (ByteBuffer) ByteBuffer.allocate(b.length).put(b).flip();
-              values.add(new String(buffer.array(), StandardCharsets.UTF_8));
-            });
+        .ifPresent(b -> values.add(Base64.getUrlEncoder().withoutPadding().encodeToString(b)));
```
Comment (brfrn169) [inline core/src/main/java/com/scalar/db/storage/cosmos/ConcatenationVisitor.java:104]: We need to use Base64 encoding to handle BLOB type for partition key.

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Review by thongdk8 — APPROVED** [review]
LGTM! Thank you!

**Comment by brfrn169** [comment]
The cause of the integration test failure is the tests taking a long time. Although the integration test is failing, I will merge this PR. And the next PR will handle the failure. Thanks.

**Pull request merged**